supporting_content/dimensions/resourceid.md

```diff
@@ -0,0 +1,55 @@
+# Dimension: ResourceId
+
+## Example mappings
+
+| Provider  | Data set                 | Vendor column                        |
+|-----------|--------------------------|--------------------------------------|
+| AWS       | CUR                      | line_item_resource_id                |
+| GCP       | Big Query Billing Export | resource.name? resource.global_name? |
+| Microsoft | Cost details             | ResourceId                           |
+
+## Examples
+| Provider  | Scenario                                          | Pattern                                                                                          |
+|-----------|---------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| AWS       | ARN provided which includes a resource name or id | arn:partition:service:region:account-id:resource-id <br>arn:partition:service:region:account-id:resource-type/resource-id<br> arn:partition:service:region:account-id:resource-type:resource-id|
+| AWS       | Some cases, just a resource-id is provided        | I-0b21f4c4434558933 OR my-cur-bucket                                                             |
+| AWS       | Tax, API call charges etc.                        | (null)                                                                                           |
+| GCP       | Resource based                                    | //container.googleapis.com/projects/<project_id>/locations/<location>/clusters/<cluster>         |
+| GCP       | Non-resource based (e.g. network egress)          | (null)                                                                                           |
+| Microsoft | Resource group resources                          | /subscriptions/<guid>/resourceGroups/<name>/providers/<provider>/<resource-type>/<resource-name> |
+| Microsoft | Subscription resources                            | /subscriptions/<guid>/providers/<provider>/<resource-type>/<resource-name>                       |
+| Microsoft | Tenant resources                                  | /providers/<provider>/<resource-type>/<resource-name>                                            |
+| Microsoft | Purchase charges                                  | (null)                                                                                           |
```

Comment:
nit: We might want to move this down after reservation and savings plan purchases, since it's the catch-all bucket.
```suggestion
| Microsoft | Marketplace and other purchases                                  | (null)                                                                                           |
```